src/js/directives/image-list-directive.test.js

```diff
@@ -2,133 +2,131 @@
+            var elm = compileDirective(defaultTemplate, scope);
+            var ligthbox;
 
-      });
+            angular.forEach(scope.mockImages, function(url, index){
+                ligthbox = elm.find('ul li a').eq(index);
+                expect(ligthbox.attr('data-lightbox')).to.equal('image-set');
+            });
 
-      it('should set a lightbox1 databox attribute for each a that matches the path in the array', function () {
-          var elm = compileDirective(defaultTemplate, scope);
-          var ligthbox;
+        });
 
-          angular.forEach(scope.mockImages, function(url, index){
-            ligthbox = elm.find('ul li a').eq(index);
-            expect(ligthbox.attr('data-lightbox')).to.equal('image-set');
-          });
+        it('should create a IMG within the A for each image', function () {
+            var elm = compileDirective(defaultTemplate, scope);
+            expect(elm.find('ul li img').length).to.equal(scope.mockImages.length);
+        });
 
-      });
+        it('should set a SRC attribute for each IMG that matches the path in the array', function () {
+            var elm = compileDirective(defaultTemplate, scope);
+            var img;
 
-      it('should create a IMG within the A for each image', function () {
-          var elm = compileDirective(defaultTemplate, scope);
-          expect(elm.find('ul li img').length).to.equal(scope.mockImages.length);
-      });
+            angular.forEach(scope.mockImages, function(url, index){
+                img = elm.find('ul li img').eq(index);
+                expect(img.attr('src')).to.equal(url);
+            });
 
-      it('should set a SRC attribute for each IMG that matches the path in the array', function () {
-          var elm = compileDirective(defaultTemplate, scope);
-          var img;
+        });
 
-          angular.forEach(scope.mockImages, function(url, index){
-            img = elm.find('ul li img').eq(index);
-            expect(img.attr('src')).to.equal(url);
-          });
+        it('should set a class of img-responsive for each IMG that matches the path in the array', function () {
+            var elm = compileDirective(defaultTemplate, scope);
+            var imgClass;
 
-      });
+            angular.forEach(scope.mockImages, function(url, index){
+                imgClass = elm.find('ul li img').eq(index);
+                expect(imgClass.attr('class')).to.equal('img-responsive');
+            });
 
-      it('should set a class of img-responsive for each IMG that matches the path in the array', function () {
-        var elm = compileDirective(defaultTemplate, scope);
-        var imgClass;
-
-        angular.forEach(scope.mockImages, function(url, index){
-            imgClass = elm.find('ul li img').eq(index);
-            expect(imgClass.attr('class')).to.equal('img-responsive');
         });
 
-      });
-
-  });
-
-});
+    });
 
+});
```

Comment:
These changes shouldn't be part of this PR. Please remove and create an additional PR if these changes are required.